cashctrl_ledger/tests/test_tax_codes.py

```diff
@@ -2,44 +2,59 @@
 
 import pytest
 import pandas as pd
-from pyledger.tests import BaseTestVatCode
+from pyledger.tests import BaseTestTaxCodes
 # flake8: noqa: F401
-from base_test import initial_ledger
+from base_test import initial_engine
+
+
+class TestTaxCodes(BaseTestTaxCodes):
+    ACCOUNTS = BaseTestTaxCodes.ACCOUNTS.copy()
+    # Set the default root node for CashCtrl. In CashCtrl it is not possible to create root nodes
+    ACCOUNTS.loc[:, "group"] = "/Assets"
+    # TODO: Remove when Assets will be implemented
+    ACCOUNTS.loc[ACCOUNTS["currency"] == "JPY", "currency"] = "USD"
+
+    TAX_CODES = BaseTestTaxCodes.TAX_CODES.copy()
+    # In CashCtrl it is not possible to create TAX CODE without specified account
+    TAX_CODES = TAX_CODES[~(TAX_CODES["id"] == "EXEMPT")]
 
 
-class TestVatCodes(BaseTestVatCode):
     @pytest.fixture(scope="class")
-    def ledger(self, initial_ledger):
-        initial_ledger.clear()
-        return initial_ledger
+    def engine(self, initial_engine):
+        initial_engine.clear()
+        return initial_engine
+
+    def test_tax_codes_accessor_mutators(self, engine):
+        super().test_tax_codes_accessor_mutators(engine, ignore_row_order=True)
 
     @pytest.mark.skip(reason="Cashctrl allows creating duplicate VAT codes")
-    def test_create_already_existed_raise_error(self):
+    def test_create_existing__tax_code_raise_error(self):
```

Comment:
Are the two adjacent underscores `__` intentional?

Reply:
Alternative possibility to deal with cash ctrl accepting duplicate tax codes:
Check for duplicates ourselves and raise an error in `tax_codes.add()`.

(Not important for me, you can also leave it as is.)